tox.ini

```diff
@@ -0,0 +1,5 @@
+[tox]
+envlist = py27,py36,py37
```

Comment:
https://github.com/revdotcom/revai-python-sdk/blob/master/.travis.yml Kevin implemented 3.4 and 3.5 as well